apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -16,11 +16,15 @@ import React, { PureComponent } from 'react';
 
 import DaySelector from './DaySelector';
 import ScheduleSelector from './ScheduleSelector';
+import LocationSelector from './LocationSelector';
 import { addCustomEvent, editCustomEvent } from '$actions/AppStoreActions';
 import analyticsEnum, { logAnalytics } from '$lib/analytics';
 import { isDarkMode } from '$lib/helpers';
 import AppStore from '$stores/AppStore';
 
+// import MapMarker from '../../../RightPane/Map/MapMarker';
```

Comment:
If you accidentally left this comment in here from trying different stuff in development, can you remove it?

On the other hand, if there is a good reason for leaving it, include that reason as another line comment.

Reply:
Yes, I accidentally left this comment. I will delete it.